tests/test_signature_coherence.py

```diff
@@ -1,31 +0,0 @@
-from crashsimilarity import crash_similarity
-
-
-if __name__ == '__main__':
```

Comment:
These can actually be useful for manual testing when we make large changes. Please don't remove them. You don't need to update them now for the new class structure, but let's leave them here.